torch/csrc/distributed/c10d/ProcessGroupWrapper.hpp

```diff
@@ -111,6 +111,11 @@ class TORCH_API ProcessGroupWrapper : public ProcessGroup {
   c10::intrusive_ptr<ProcessGroup::Work> barrier(
       const BarrierOptions& opts = BarrierOptions()) override;
 
+    c10::intrusive_ptr<ProcessGroup::Work> _reduce_scatter_base(
+      at::Tensor& outputBuffer,
+      at::Tensor& inputBuffe,
```

Comment:
`inputBuffer`? (wonder why compiler doesn't complain as it mismatches the cpp argument)

Reply:
Fixed. arg names have no meaning in forward declarations.